### PR TITLE
Translate several release notes to Simplified Chinese (zh_cn)

### PR DESCRIPTION
**Pull request by dsh0416**

### CODE REVIEW

**Comment by hlcfan** [comment]
Thank you!

File: zh_cn/news/_posts/2016-12-25-ruby-2-4-0-released.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+Ruby 在线程等待地时候会进行死锁检查，但是检查的结果没有足够的信息用来调试。
+Ruby 2.4 死锁检查会显示他们的栈信息和依赖线程。
+
+## 其他自 2.3 起显著的改变
+
+* 支持 OpenSSL 1.1.0 版本（对 0.9.7 及之前版本不再支持）
+* ext/tk 现已从标准库中移除 [Feature #8539](https://bugs.ruby-lang.org/issues/8539)
+* XMLRPC 现已从标准库中移除 [Feature #12160](https://bugs.ruby-lang.org/issues/12160)
+
+请参阅 [NEWS](https://github.com/ruby/ruby/blob/v2_4_0/NEWS)。
```
Comment (hlcfan) [inline zh_cn/news/_posts/2016-12-25-ruby-2-4-0-released.md]: Seems you missed `or commit logs for details.` here.

**Comment by dsh0416** [comment]
@hlcfan Problems above fixed in the latest commit.

**Comment by hlcfan** [comment]
Looks good to me, @JuanitoFatas Please have a 👀

**Comment by AndorChen** [comment]
@dsh0416 Thanks for your translation. But DO NOT modify other language next time, our focus is Simplified Chinese.

@hlcfan Thanks for your review.

**Comment by stomar** [comment]
@AndorChen @dsh0416 @hlcfan 

PR's **must not include any merge commits**. I reverted to a prior commit.

@dsh0416
Please provide a new, clean PR (and a separate PR for the zh_tw changes).

**Comment by stomar** [comment]
Update: I committed the translations for `zh_cn`, see 1aabcdc0b71147a456b0c72d5bafe0cd5d1eff26.

**Comment by dsh0416** [comment]
@stomar OK, I would provide a separate clean `zh_tw` patch in minutes.

**Comment by dsh0416** [comment]
@stomar The separate PR for zh_tw changes has submitted [PR#1540](https://github.com/ruby/www.ruby-lang.org/pull/1540)

**Comment by AndorChen** [comment]
@stomar Got it.

**Pull request merged**